GTG/plugins/unmaintained/bugzilla/services.py

```diff
@@ -8,6 +8,17 @@
 
 __all__ = ('BugzillaServiceFactory',)
 
+# Register bugzilla services manually, however store them in someplace and load
+# them at once is better.
+services = {
+    'bugzilla.gnome.org': GnomeBugzilla,
+    'bugs.freedesktop.org': FreedesktopBugzilla,
+    'bugzilla.mozilla.org': MozillaBugzilla,
+    'bugzilla.samba.org': SambaBugzilla,
+    'bugs.gentoo.org': GentooBugzilla,
+    'bugzilla.redhat.com': RedHatBugzilla,
+}
+
```

Comment:
While the plugin is not maintained, this shouldn't work because the referenced classes are declared below, so python wouldn't know about it at that point and fail.

Reply:
Was not paying attention...will put this back